bin/testRelease

```diff
@@ -1,145 +0,0 @@
-#! /bin/bash
-
-#################### Running this Script #######################
-# Verify Git status is clean
-# Run this with `./bin/testRelease`
-################################################################
-
-SOMETHING_FAILED=0
-echo "#########################################################"
-echo '# Test a release of Ignite before actually releasing it #'
-echo '#                  ._______.                            #'
-echo '#                   | \   / |                           #'
-echo '#                .--|.O.|.O.|______.                    #'
-echo '#              __).-| = | = |/   \ |                    #'
-echo "#              >__) (.'---\`.)Q.|.Q.|--.                 #"
-echo '#                    \\___// = | = |-.(__               #'
-echo "#                     \`---'( .---. ) (__<               #"
-echo "#                           \\\\.-.//                     #"
-echo "#                            \`---'                      #"
-echo '#########################################################'
```

Comment:
Re-introduced in #1011 @GantMan. It seemed to duplicate what we already have in CI, but there are some additional steps in it, so it's worth keeping.